recipes/pyusb/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "pyusb" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4e9b72cc4a4205ca64fbf1f3fff39a335512166c151ad103e55c8223ac147362
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

Conda-build does most of this for us these days. So this can just be the following...

```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```